File: lang-kotlin/src/test/resources/fixtures/parser/kotlin-loops.java
import java.util.List;

class ItemProcessor {
    void processAll(List items) {
        items.forEach(item -> {
            System.out.println(item);
        });
    }

    void filterItems(List items, List targets) {
        items.forEach(item -> {
            if (targets.contains(item)) {
                System.out.println(item);
            }
        });
    }

    void sortAndGet(List items) {
        items.sort((a, b) -> a.toString().compareTo(b.toString()));
    }
}

File: lang-kotlin/src/test/resources/fixtures/parser/kotlin-loops.kt
class ItemProcessor {
    fun processAll(items: List<String>) {
        items.forEach { item ->
            println(item)
        }
    }

    fun filterItems(items: List<String>, targets: List<String>) {
        items.forEach { item ->
            if (targets.contains(item)) {
                println(item)
            }
        }
    }

    fun sortAndGet(items: MutableList<String>) {
        items.sort()
    }
}
